tests/test_validators.py

```diff
@@ -515,21 +515,25 @@ def test_in_all(self):
         """
         assert deep_iterable.__name__ in validator_module.__all__
 
-    def test_success_member_only(self):
+    @pytest.mark.parametrize(
+        "member_validator", [instance_of(int), [always_pass, instance_of(int)]]
+    )
```

Comment:
Your using this a lot; please transform it into a [parametrized fixture](https://hynek.me/til/parametrized-pytest-fixtures/).

Reply:
This is exactly what I was looking for, Nice blog post!

Reply:
my review took a day longer because i wanted to write it up properly ;)